pgml-extension/src/bindings/transformers/mod.rs

```diff
@@ -389,3 +408,88 @@ pub fn clear_gpu_cache(memory_usage: Option<f32>) -> Result<bool> {
         Ok(success)
     })
 }
+
+// Called before hugginface python APIs. Setup ENVs for HuggingFace. See
+// https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#hfhuboffline
+pub fn set_env() {
+    let envs_to_apply = guc::gen_hf_env_map();
+    let py_inited = unsafe { ffi::Py_IsInitialized() != 0 };
```

Comment:
This works with PyO3? We've made sure that PyO3 is loaded into local area in our .so: https://github.com/postgresml/postgresml/blob/master/pgml-extension/ld.map#L7